functions/controllers/user/requestFriend.js

```diff
@@ -0,0 +1,84 @@
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');
+const { message, status } = require('../../util/constants');
+
+const { error, success } = status;
+const { somethingWentWrong, userNotFound } = message;
+
+const errors = {
+    error1: res => {
+        res.status(BAD_REQUEST).json({ message: 'cannot follow yourself' });
+    },
+    error2: res => {
+        res.status(INTERNAL_SERVER_ERROR).json({
+            message: somethingWentWrong,
+            status: error,
+        });
+    },
+    error3: res => {
+        res.status(BAD_REQUEST).json({
+            message: userNotFound,
+            status: error,
+        });
+    },
+    error4: res => {
+        res.status(BAD_REQUEST).json({
+            message: 'Request Already sent Awaitting Acceptance',
+            status: error,
+        });
+    },
+};
```

Comment:
keep your code looking consistent. `line 9` is inconsistent with the rest of the errors.
Why are some messages in variables and others in strings?